app/controllers/episodes_controller.rb

```diff
@@ -134,6 +134,14 @@ def episode_params
       :original_guid,
       categories: [],
       images_attributes: %i[id original_url size alt_text caption credit _destroy _retry]
-    )
+    ).tap do |p|
+      p[:released_at] = released_at_zone.parse(p[:released_at]) if p[:released_at].present?
```

Comment:
trying to understand this: if someone has chosen a release date, we need to also check the time zone that was selected, and then parse the final date off of that zone using the release date given.